### PR TITLE
File uploader modal styling complete.

### PR DESCRIPTION
**Pull request by sgortz**
Added CSS to the modal to match the wireframe. Added a comment for @Skoomatron with instructions to add code to the Homepage.  Still working on the drag n' drop feature, but nothing is broken, so it's safe to merge.

### CODE REVIEW

**Comment by sgortz** [comment]
Also, I'm using hooks to toggle the modal on/off but I'm down to start using Recoil for this!

File: src/App.js [hunk 1]
```diff
@@ -4,17 +4,22 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import { signInWithGoogle } from './auth/Firebase';
 import Header from './components/Header.jsx'
 import Landing from './components/Landing/Landing.jsx'
+import FileUploaderModal from "./components/file-uploader-modal/FileUploaderModal.jsx";
 
-function App () {
+function App() {
   const [page, setPage] = useState('landing')
+  const [show, setShow] = useState(false)
```
Comment (sgortz) [inline src/App.js:11]: @Skoomatron Line 11 also to be added to `Homepage` and be deleted from `App.js`

**Review by Acid-Override — COMMENTED** [review]
Awesome work here Sabrina!

**Comment by sgortz** [comment]
Also, I ended up pushing the commit with a new dependency. Sorry guys! Should I close the PR revert back before pushing again?

**Review by Acid-Override — APPROVED** [review]

**Comment by Acid-Override** [comment]
> Also, I ended up pushing the commit with a new dependency. Sorry guys! Should I close the PR, and revert back before pushing again?

It shouldn't matter as long as we all do a "npm ci" after we merge the changes.

**Pull request closed without merging**